hack/cleanup/resource_mgt.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/digitalocean/godo"
+
+	"sigs.k8s.io/cluster-api-provider-digitalocean/test/e2e"
+)
+
+func main() {
+	log.Println("Starting DO Janitor")
+	token := os.Getenv("DIGITALOCEAN_ACCESS_TOKEN")
+	if token == "" {
+		log.Fatal("missing DO token")
+	}
+
+	ctx := context.Background()
+	client := godo.NewFromToken(token)
+
+	today := time.Now().UTC()
+
+	droplets, err := e2e.DropletList(ctx, client)
+	if err != nil {
+		log.Fatalf("fail to list droplets: %+v", err.Error())
+	}
+
+	for _, droplet := range droplets {
+		dropletCreated, err := time.Parse(time.RFC3339, droplet.Created)
+		if err != nil {
+			log.Fatalf("failt parsing time: %+v", err.Error())
+		}
+
+		days := today.Sub(dropletCreated).Hours() / 24
+		if days >= 2 {
+			log.Printf("%s is older than 2 day will terminate\n", droplet.Name)
+			resp, err := client.Droplets.Delete(ctx, droplet.ID)
+			if err != nil {
+				log.Printf("fail to delete droplet %s: %+v\n", droplet.Name, err.Error())
+				continue
+			}
+			if resp.StatusCode != http.StatusNoContent {
```

Comment:
The way godo works is that it [always returns a non-nil error if the response code is > 300](https://github.com/digitalocean/godo/blob/05dc4d920aacfcded4808de4d5b0d7bb78793606/godo.go#L436-L451). Unless we care about a specific 2xx code here, I think we could also drop this status code check.

Reply:
dropped